### ISSUE TITLE
Change alert colors back

### ISSUE THREAD

**Issue by iamthestreets** [designmodo/Flat-UI-Pro-Support#518]
How do I change the alerts back to the original colors? The old style is much better then these.

**Comment by iamthestreets** [comment]
Any help with this?

**Comment by andrew-ml** [comment]
Hello, @iamthestreets, 
what do you mean by the old style colors? 
Do you mean Bootstrap original colors?  http://getbootstrap.com/javascript/#alerts

**Comment by iamthestreets** [comment]
yes. The new style is not good IMO. You can barely see what color it is and doesn't really match the rest of the styles. I was able to modify it myself, but it would be nice if the default style for this was a little different.
